pennylane/ops/op_math/decompositions/solovay_kitaev.py

```diff
@@ -73,8 +73,8 @@ def _contains_SU2(op_mat, ops_vecs=None, kd_tree=None, tol=1e-8):
         tol (float): Tolerance for the match to be considered ``True``.
 
     Returns:
-        Tuple(bool, TensorLike, int): A bool that shows whether an operation similar to the given operations
-        was found, the quaternion representation of the searched operation and its index in the list.
+        Tuple(bool, TensorLike, int): A bool that shows whether an operation similar to the given operations was
+        found, the quaternion representation of the searched operation and its index in the `op_vecs` or `kd_tree`.
```

Comment:
```suggestion
        Tuple(bool, TensorLike, int): A tuple including `True`/`False` for whether an operation similar to the given operations was
        found, the quaternion representation of the searched operation, and its index in the `op_vecs` or `kd_tree`.
```

Reply:
I was initially a bit confused reading this. So I just added a bit of clarification :)

Reply:
Oh! This is an internal function lol. Well, can't hurt! hahaha